### DISCUSSION TITLE
how to take enb.log

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#230] [General]
hi I trying to see enb log . the enb.config i am seeing path of enb log   **filename = /mnt/srslte/enb.log**

but suppose i will see inside /mnt folder srsrlte folder is not available .

how to take enb log

**Comment by herlesupreeth** [comment]
You should see in under srslte folder on the host machine (not in container)

**Reply by herlesupreeth** [reply]
![image](https://github.com/herlesupreeth/docker_open5gs/assets/4115839/23d7e0a9-e8b1-49d4-a441-ea03fbd0d22f)

**Reply by Sivanesh1992** [reply]
thankyou lot
